Resources/Prototypes/Entities/Mobs/Species/diona.yml

```diff
@@ -10,7 +10,7 @@
   - type: Hunger
     baseDecayRate: 0.0083
```

Comment:
this should probably be increased a bit so its ~half instead of 1/6 which is a huge disparity

Reply:
Sure thing.